README.md

```diff
@@ -13,6 +13,7 @@ To view the loom introductory video click here: [Live Introduction](https://www.
 - Html
 - CSS
 - Javascript
+- Github
 
 ## Installation
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As mentioned by the previous reviewer, kindly make sure you update your README by using the new template lunched by Microverse. Here is the [template](https://github.com/microverseinc/readme-template)